services/brig/src/Brig/App.hs

```diff
@@ -400,29 +405,56 @@ initHttp2Manager = do
 -- faster. So, we reuse the context.
 
 -- TODO: somewhat duplicates Galley.App.initExtEnv
-initExtGetManager :: IO (Manager, [Fingerprint Rsa] -> SSL.SSL -> IO ())
-initExtGetManager = do
-  ctx <- SSL.context
-  SSL.contextAddOption ctx SSL_OP_NO_SSLv2
-  SSL.contextAddOption ctx SSL_OP_NO_SSLv3
-  SSL.contextSetCiphers ctx rsaCiphers
-  -- We use public key pinning with service providers and want to
-  -- support self-signed certificates as well, hence 'VerifyNone'.
-  SSL.contextSetVerificationMode ctx SSL.VerifyNone
-  SSL.contextSetDefaultVerifyPaths ctx
-  mgr <-
+initExtGetManager :: IO ([Fingerprint Rsa] -> IO Manager)
+initExtGetManager =
+  pure $ \fingerprints -> do
```

Comment:
this is the only thing that makes it `IO`, should probably remove it and lift it at the appropriate place.